packages/mui-types/index.d.ts

```diff
@@ -146,3 +146,10 @@ export interface OverridableTypeMap {
  */
 // tslint:disable-next-line: ban-types
 export type Simplify<T> = T extends Function ? T : { [K in keyof T]: T[K] };
+
+/**
+ * Changes the properties K from T to required
+ */
+export type PickRequired<T, K extends keyof T> = DistributiveOmit<T, K> & {
+  [P in K]-?: T[P];
```

Comment:
what does `-` do here?

Reply:
@sai6855 This changes an optional attribute to a required one: https://www.typescriptlang.org/docs/handbook/2/mapped-types.html#mapping-modifiers

Reply:
Nice!! Didn't knew this